Upcoming version 0.16.0 preview enhancements (#1057)
microsoftgraph/entra-powershell: module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaApplicationOwner.md

- `-ObjectId` parameter specifies the unique identifier of an application.

> ### Example 2: Get the owner of an application with more owner details

Author: ```suggestion
### Example 2: Get the details about the owner of an application
```
